src/test/java/seedu/address/testutil/TypicalTags.java

```diff
@@ -0,0 +1,13 @@
+package seedu.address.testutil;
+
+import seedu.address.model.tag.Tag;
+
+/**
+ * A utility class containing a list of {@code Test} objects to be used in tests.
+ */
+public class TypicalTags {
```

Comment:
Personally, I think it would be better to combine the `TypicalTags` and `TypicalTagNames` since they are pretty much the same thing, although this is just a casual opinion from me!

However, I think we should use the constants we defined in `TypicalTagNames.java` instead of re-defining the literals here, for example ` VALID_TAG_FRIENDS = new Tag(VALID_TAGNAME_FRIENDS)`